src/components/ChatSidebar/ChatSidebar.tsx

```diff
@@ -0,0 +1,46 @@
+"use client";
+
+import {
+  Sidebar,
+  SidebarContent,
+  SidebarGroup,
+  SidebarGroupContent,
+  SidebarGroupLabel,
+  SidebarHeader,
+  SidebarMenu,
+} from "@/components/ui/sidebar";
+import { Button } from "@/components/ui/button";
+import { NotebookPen, Star } from "lucide-react";
+import { ChatSidebarTrigger } from "@/components/ChatSidebar/ChatSidebarTrigger";
+import { ChatSidebarLink } from "@/components/ChatSidebar/ChatSidebarLink";
+
+export function ChatSidebar() {
+  return (
+    <Sidebar>
+      <SidebarHeader className="flex-row justify-between px-4 pt-4 pb-0">
+        <ChatSidebarTrigger />
+        <div className="flex gap-2">
+          <Button size="icon" variant="transparent" title="Zamknij menu">
+            <Star size={24} />
+          </Button>
+          <Button size="icon" variant="transparent" title="Zamknij menu">
```

Comment:
Titles of buttons are leftovers from previous implementation I guess

Reply:
yup, forgot to change them 😭